scrapers/Brazzers.yml

```diff
@@ -23,11 +49,154 @@ xPathScrapers:
+              MT: "USA"
+              NC: "USA"
+              ND: "USA"
+              NE: "USA"
+              NH: "USA"
+              NJ: "USA"
+              NM: "USA"
+              NV: "USA"
+              NY: "USA"
+              OH: "USA"
+              OK: "USA"
+              OR: "USA"
+              PA: "USA"
+              RI: "USA"
+              SC: "USA"
+              SD: "USA"
+              TN: "USA"
+              TX: "USA"
+              UT: "USA"
+              VA: "USA"
+              VT: "USA"
+              WA: "USA"
+              WI: "USA"
+              WV: "USA"
+              WY: "USA"              
+              Alabama: "USA"
+              Alaska: "USA"
+              Arizona: "USA"
+              Arkansas: "USA"
+              California: "USA"
+              Colorado: "USA"
+              Connecticut: "USA"
+              Delaware: "USA"
+              Florida: "USA"
+              Georgia: "USA"
+              Hawaii: "USA"
+              Idaho: "USA"
+              Illinois: "USA"
+              Indiana: "USA"
+              Iowa: "USA"
+              Kansas: "USA"
+              Kentucky: "USA"
+              Louisiana: "USA"
+              Maine: "USA"
+              Maryland: "USA"
+              Massachusetts: "USA"
+              Michigan: "USA"
+              Minnesota: "USA"
+              Mississippi: "USA"
+              Missouri: "USA"
+              Montana: "USA"
+              Nebraska: "USA"
+              Nevada: "USA"
+              New Hampshire: "USA"
+              New Jersey: "USA"
+              New Mexico: "USA"
+              New York: "USA"
+              North Carolina: "USA"
+              North Dakota: "USA"
+              Ohio: "USA"
+              Oklahoma: "USA"
+              Oregon: "USA"
+              Pennsylvania: "USA"
+              Rhode Island: "USA"
+              South Carolina: "USA"
+              South Dakota: "USA"
+              Tennessee: "USA"
+              Texas: "USA"
+              Utah: "USA"
+              Vermont: "USA"
+              Virginia: "USA"
+              Washington: "USA"
+              West Virginia: "USA"
+              Wisconsin: "USA"
+              Wyoming: "USA"
+      Height:
+        selector: //*[contains(.,"Height")]/following-sibling::span/text()[not(contains(.,"N/A"))]
+        postProcess:
+          - feetToCm: true
+      Image:
+        selector: //img[contains(@src, "model")]/@src
+      Ethnicity:
+        selector: //*[contains(.,"Ethnicity")]/following-sibling::a/text()
+      FakeTits:
+        selector: //a[contains(@href,"/enhanced")]/text()|//a[contains(@href,"/natural-tits")]/text()
+        postProcess:
+          - map:
+              Natural Tits: "No"
+              Enhanced: "Yes"
+      Piercings:
+        selector: //*[contains(.,"Body Art")]/following-sibling::a[contains(@href,"/piercing")]/text()
+        postProcess:
+          - map:
+              Piercing: "Yes"
+# There is no Xpath to know the gender.
```

Comment:
Like piercings, tattoos can also be extracted (sample `https://www.brazzers.com/pornstar/6671/brooklyn-chase`)
```yaml
      Tattoos:
        selector: //*[contains(.,"Body Art")]/following-sibling::a[contains(@href,"/tattoo")]/text()
        postProcess:
          - map:
              Tattoo: "Yes"
```

Reply:
I searched some performer that could have this tags and didn't found it. (Like Bonnie Rotten, have lot of tattoos but not the tags...)